### PR TITLE
[SONAR-6926] remove Cross Project Duplication properties declaration until 5.3

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

File: sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java [hunk 1]
```diff
@@ -262,40 +262,40 @@ private CorePropertyDefinitions() {
...
-        .defaultValue("")
-        .name("Duplication Exclusions")
-        .description("Patterns used to exclude some source files from the duplication detection mechanism. " +
-          "See below to know how to use wildcards to specify this property.")
-        .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
-        .category(CoreProperties.CATEGORY_EXCLUSIONS)
-        .subCategory(CoreProperties.SUBCATEGORY_DUPLICATIONS_EXCLUSIONS)
-        .multiValues(true)
-        .build()));
+//      PropertyDefinition.builder(CoreProperties.CPD_CROSS_PROJECT)
```
Comment (julienlancelot) [inline sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java]: Please just add a comment why this property is removed and a link to the ticket that will reactive the feature.

**Comment by julienlancelot** [comment]
On minor feedback and LGTM

**Comment by julienlancelot** [comment]
And don't forget to merge on branch-5.2 and not on master

**Comment by sns-seb** [comment]
merged to branch "branch-5.2", closing PR manually

**Pull request closed without merging**